Purchases/Purchasing/Package.swift

```diff
@@ -36,6 +36,9 @@ import Foundation
          weekly
 }
 
+/**
+ Enumeration of all possible Package types.
```

Comment:
I think you meant this in the `public enum` above?

Reply:
haha yep! I saw that you fixed this in the other PR so I'm just merging for now